WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/WatchFaceRepository.kt

```diff
@@ -34,42 +33,32 @@ class WatchFaceRepository(
 ) {
 
     // [AnalogWatchFaceEntity] properties/operations:
-    val allAnalogWatchFaces: Flow<List<AnalogWatchFaceEntity>> = analogWatchFaceDao.getAll()
+    suspend fun getAnalogWatchFace(analogWatchFaceId: Int) =
+        analogWatchFaceDao.get(analogWatchFaceId)
 
-    @WorkerThread
-    suspend fun insertAnalogWatchFace(analogWatchFace: AnalogWatchFaceEntity) {
-        analogWatchFaceDao.insert(analogWatchFace)
-    }
+    fun getAnalogWatchFaceAndStylesAndDimensions(analogWatchFaceId: Int) =
+        analogWatchFaceDao.getAnalogWatchFaceAndStylesAndDimensions(analogWatchFaceId)
 
     @WorkerThread
-    suspend fun deleteAllAnalogWatchFaces() {
-        analogWatchFaceDao.deleteAll()
+    suspend fun updateAnalogWatchFace(analogWatchFace: AnalogWatchFaceEntity) {
+        analogWatchFaceDao.update(analogWatchFace)
     }
 
     // [WatchFaceColorStyleEntity] properties/operations:
-    val watchFaceColorStyle: Flow<List<WatchFaceColorStyleEntity>> = watchFaceColorStyleDao.getAll()
-
     @WorkerThread
-    suspend fun insertWatchFaceColorStyle(colorStyle: WatchFaceColorStyleEntity) {
-        watchFaceColorStyleDao.insert(colorStyle)
-    }
+    suspend fun getAllWatchFaceColorStyles(): List<WatchFaceColorStyleEntity> =
+        watchFaceColorStyleDao.getAll()
 
     @WorkerThread
-    suspend fun deleteAllWatchFaceColorStyles() {
-        watchFaceColorStyleDao.deleteAll()
-    }
+    suspend fun getWatchFaceColorStyles(watchFaceColorStylesId: String): WatchFaceColorStyleEntity =
+        watchFaceColorStyleDao.get(watchFaceColorStylesId)
 
     // [WatchFaceArmDimensionsEntity] properties/operations:
-    val watchFaceArmDimensions: Flow<List<WatchFaceArmDimensionsEntity>> =
-        watchFaceArmDimensionsDao.getAll()
-
-    @WorkerThread
-    suspend fun insertWatchFaceArmDimensions(armDimensions: WatchFaceArmDimensionsEntity) {
-        watchFaceArmDimensionsDao.insert(armDimensions)
-    }
+    suspend fun getWatchFaceArmDimensions(watchFaceArmDimensionsId: String) =
+        watchFaceArmDimensionsDao.get(watchFaceArmDimensionsId)
 
     @WorkerThread
-    suspend fun deleteAllWatchFaceArmDimensions() {
-        watchFaceArmDimensionsDao.deleteAll()
+    suspend fun updateWatchFaceArmDimensions(armDimensions: WatchFaceArmDimensionsEntity) {
+        watchFaceArmDimensionsDao.update(armDimensions)
```

Comment:
For most methods, you are using an assignment instead of brackets. Why is this one different?